addons/exact-count/addon.json

```diff
@@ -20,6 +20,12 @@
       "id": "user",
       "type": "boolean",
       "default": true
+    },
+    {
+      "name": "Exact count for studios",
+      "id": "studio",
```

Comment:
Yes, this is the same setting ID that was removed in #7617, so it's like we never removed the setting 👍

Reply:
The last value of the "forumuser" setting will not be stored anywhere, so if we ever readd that feature somehow, users will need to enable it again.